docs/community.md

```diff
@@ -0,0 +1,67 @@
+# Community
+
+Jupyter Book is an **open source project** with a **diverse community of contributors and leaders**. It is a sub-project of [Project Jupyter](https://jupyter.org). Here are a few ways that you can engage with our community.
+
+## Where to talk
+
+::::{grid} 3
+:::{card} Contributor and user Discord 💬
+:link: https://discord.mystmd.org
+For general chat in the Jupyter Book and MyST community
++++
+Click to visit the Discord →
+:::
+:::{card} Discussions Forum 💬
+:link: https://github.com/jupyter-book/mystmd/discussions
+For general conversation about the MyST Markdown engine.
++++
+Click to visit the Discussion Forum →
+:::
+:::{card} Events calendar 🗓️
+:link: https://compass.jupyterbook.org/contribute#events
+Connect with others in the Jupyter Book community in real-time.
++++
+Click to see our events calendar →
+:::
+::::
+
+## Where to learn
+
+::::{grid} 2
+:::{card} The Jupyter Book User Guide
+:link: ./start.md
+Learn how to use Jupyter Book.
++++
+Click to visit the Jupyter Book User Guide →
+:::
+:::{card} The MyST Document Engine Guide
```

Comment:
```suggestion
:::{card} The MyST Document Engine guide
```

Reply:
I'm not sure yet whether MyST Document Engine or MyST document engine feels more natural. I'm leaning towards "it's a name".

Reply:
If we make it all part of the name then we can have a tag-line like "You down with MDE yeah you know me!"